tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{38,39}-django{32,40}
+    py{38,39,312}-django{32,40}
```

Comment:
[Ona Data](https://github.com/onaio/onadata) tests on 3.10. Can we drop `38` and `39`? Can we add 3.10 and perhaps 3.11?

Reply:
done!